src/prometheus_alert_rules/accept_login_request_failed.rule

```diff
@@ -0,0 +1,6 @@
+alert: AcceptLoginRequestFailed
+expr: sum(increase(http_response_time_seconds_count{route="GET/api/kratos/self-service/login/browser", status="500"}[2m])) > 0
```

Comment:
I would say that we could group by route (exclude healthcheck and metrics maybe) and cater for status =~5.*

you would get all the rules for each endpoint in one

Reply:
done, also changed it in the dashboard